enterprise/ha/src/test/java/jmx/HaBeanIT.java

```diff
@@ -68,7 +71,22 @@
     public static final ClusterRule clusterRule = new ClusterRule( HaBeanIT.class )
             .withInstanceSetting( setting( "jmx.port", STRING, (String) null ), intBase( 9912 ) )
             .withInstanceSetting( HaSettings.ha_server, stringWithIntBase( ":", 1136 ) )
-            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) );
+            .withInstanceSetting( GraphDatabaseSettings.forced_kernel_id, stringWithIntBase( "kernel", 0 ) )
+            .withStoreDirInitializer( new ClusterManager.StoreDirInitializer()
```

Comment:
Would it make sense to have ClusterRule do this operation? Currently it uses a directory based off the class/method name, without cleaning it. I assume no tests would break if cleaning was part of initializing the cluster rule.